blog/2021-03-01-OptimizingCovid19VaccinationScheduling.adoc

```diff
@@ -0,0 +1,145 @@
+= Optimizing COVID-19 vaccination appointment scheduling
+:page-interpolate: true
+//:imagesdir: 2021/03/01
+:awestruct-author: pbrown and emmurphy
```

Comment:
This won't work unfortunately :( Our current system only supports one author (yes, it's unfair, yes we need to fix it, but we can't right now).

Suggestion: just mention "pbrown" only and in the text at the bottom add "Co-authored by Emily Murphy". Reverse it for the second blog.